### ISSUE TITLE
The pretrained weight

### ISSUE THREAD

**Issue by RealStar1** [open-mmlab/mmdetection#9017]
we want to built a custom detector  for our dataset.

we use the Mask-RCNN as the benchmark. we use GPU =1 、 batch_size= 6 、opetimizer = SGD、 learning rate = 0.02、max_epoch = 60 as the  training parameters.

we find that whether or not  the pre-trained weight on imagenet was used had a significant impact for the experimental results. 
There are 20 percent gap on the map50  and AP50 values in bbox and segm task.

So  what we should do in the next step.  the custom models  do not have a proper pre-trained weights.

**Comment by RangiLyu** [comment]
The default config of Mask-RCNN with ResNet50 freezes the BN layers' mean&std and the first backbone stage.
```python
    backbone=dict(
        type='ResNet',
        depth=50,
        num_stages=4,
        out_indices=(0, 1, 2, 3),
        frozen_stages=1,  # freeze the first backbone stage, set to -1 means not freeze
        norm_cfg=dict(type='BN', requires_grad=True),
        norm_eval=True,  # freeze the BN layers' mean&std, set to False means not freeze
        style='pytorch',
        init_cfg=dict(type='Pretrained', checkpoint='torchvision://resnet50')),
```
If you want to train with your own model, you should not freeze the parameters.

Also, you need longer training epochs if you do not have a pretrained weight.
You can refer to `configs/scratch/README.md` and the paper [Rethinking ImageNet Pre-training](https://arxiv.org/abs/1811.08883)

**Comment by RealStar1** [comment]
i am very appreciate for you response,

**Comment by RealStar1** [comment]
> The default config of Mask-RCNN with ResNet50 freezes the BN layers' mean&std and the first backbone stage.
> 
> ```python
>     backbone=dict(
>         type='ResNet',
>         depth=50,
>         num_stages=4,
>         out_indices=(0, 1, 2, 3),
>         frozen_stages=1,  # freeze the first backbone stage, set to -1 means not freeze
>         norm_cfg=dict(type='BN', requires_grad=True),
>         norm_eval=True,  # freeze the BN layers' mean&std, set to False means not freeze
>         style='pytorch',
>         init_cfg=dict(type='Pretrained', checkpoint='torchvision://resnet50')),
> ```
> 
> If you want to train with your own model, you should not freeze the parameters.
> 
> Also, you need longer training epochs if you do not have a pretrained weight. You can refer to `configs/scratch/README.md` and the paper [Rethinking ImageNet Pre-training](https://arxiv.org/abs/1811.08883)

i am very appreciate for you response,
